test/mocha-loader-hook.js

```diff
@@ -0,0 +1,20 @@
+//mocha = require("mocha")
+//
+//// Stash a reference away to this
+//old_loader = mocha.prototype.loadFiles
+//
```

Comment:
This was added because mocha would just exit silently when the coffee-script had syntax errors. Very annoying!